src/libmscore/skyline.cpp

```diff
@@ -284,6 +284,12 @@ void SkylineLine::paint(QPainter& p) const
         }
         x1 = x2;
     }
+
```

Comment:
still a line too much for my taste ;-)

Reply:
`uncrustify` had the same comment :wink:

Reply:
@jojo-bot at work ;-)

Reply:
And I thought giving the bot some extra work :smile: